Configuration/NodeTypes.yaml

```diff
@@ -256,7 +256,8 @@
         reloadIfChanged: true
         inspector:
           group: options
-
+    value:
+      defaultValue: 'Checked'
```

Comment:
ist das hier korrekt? Bei den anderen `type: boolean`s hat man als Default Value auch true/false, hier `Checked`. Also entweder ist type falsch oder dieses defaultValue, oder? Was sagt die Spezifikation dazu?

Reply:
@remuslazar das value ist eine extra property und bezieht sich nicht auf das checked.
Das Value dient zur valideren und darf nicht null sein und wird in der späteren Mail erscheinen.
Also label:value.  Siehe Kommentare https://cron-eu.atlassian.net/browse/DAVSHOP-891

Reply:
ach so, ok. Das ist aber schon Irre führend, warum hat man das einfach so "value" genannt? Gäbe es keinen besseren, sinnigeren Namen dafür? Aber gut, muss man jetzt nicht alles umbauen deswegen.

Reply:
@remuslazar Beispiel:

```
<label>Ich akzeptiere bla bla: </label>
<input type="check" value="Checked">
```
hier:
* label = "Ich akzeptiere bla bla"
* value = "Checked"

Das erscheint dann später auch so in der Mail. Kann man je nach "Frage" die hinter der Checkbox steht auch mit was anderem Belegen z.B. "Akzeptiert", das ist ja nur ein sinnvoller generischer Default ("Checked").

Reply:
@baschny ok, danke, mir da hier ein wenig Kontext gefehlt, um das richtig zu verstehen. Ich hätte das nicht `value` genannt, um diese "Verwechselungsgefahr" vorzubeugen (weil im YAML auch "defaultValue" gibt), z.B. hätte ich `checkboxValue` oder `controlValue` favorisiert. Aber ist auch ok so und auch "kurz", alles gut.